components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -19,14 +19,26 @@ import (
 	storage "github.com/chef/automate/components/authz-service/storage/v2"
 	"github.com/chef/automate/components/authz-service/testhelpers"
 	"github.com/chef/automate/lib/grpc/auth_context"
-	uuid "github.com/chef/automate/lib/uuid4"
 )
 
 const (
 	Applied = "applied"
 	Staged  = "staged"
 )
 
+const (
+	policyMembersByMemberName = "SELECT count(*) FROM iam_policy_members WHERE member_id=member_db_id($1)"
```

Comment:
This should read  `ByMemberID` I think

Reply:
nope, the `$1` is the member name that gets resolved to a member_id (db_id)

Reply:
Oh what an awkward sql statement 😂